common/app/model/content/Atoms.scala

```diff
@@ -86,7 +87,14 @@ object Atoms extends common.Logging {
     content.atoms.map { atoms =>
       val quizzes = extract(atoms.quizzes, atom => { Quiz.make(content.id, atom) })
 
-      val media = extract(atoms.media, atom => { MediaAtom.make(atom) })
+      val media = extract(atoms.media, atom => {
+        val endSlatePath = EndSlateComponents(
+          sectionId = content.sectionId.getOrElse(""),
+          shortUrl = content.fields.flatMap(_.shortUrl).getOrElse(""),
```

Comment:
@akash1810 in this case the `endSlatePath` is set to `None` in: https://github.com/guardian/frontend/blob/14c8ef7bbc0b07374b386c5a521a3601a1fc5ebd/applications/app/controllers/MediaAtomEmbedController.scala#L22 so the class which triggers the end slate to appear will not be added.